cmd/launcher/svc_windows.go

```diff
@@ -29,6 +29,11 @@ import (
 // TODO This should be inherited from some setting
 const serviceName = "launcher"
 
+var likelyRootDirPaths = []string{
+	"C:\\ProgramData\\Kolide\\Launcher-kolide-k2\\data",
+	"C:\\Program Files\\Kolide\\Launcher-kolide-k2\\data",
```

Comment:
I _think_ this is the correct order? It means that if there's a new db path, we should prefer it. And I think that's likely to be true if someone got themselves into this situation.  (They'll have already done the pain of it, and going back to the old db will be worse)

Reply:
that was my thinking as well. I think for most (upgrade) cases moving forward we'll get ProgramData via opts anyway and check that first before we even iterate these

Reply:
Mostly talking through my assumptions here.